.travis.yml

```diff
@@ -3,12 +3,15 @@ language: ruby
 rvm:
 - 2.3.3
 
+services:
+- postgresql
+
 script:
-- rvm gemset use global
 - bundle install
-- gem uninstall fileutils
-- gem install fileutils
-- gem update fileutils
-- rails db:create
-- rails db:migrale
-
+- psql -c "create user alex with encrypted password 'qwerty123';" -U postgres
+- psql -c "CREATE DATABASE myapp_test;" -U postgres
+- psql -c "GRANT ALL PRIVILEGES ON DATABASE myapp_test to alex;" -U postgres
+- psql -c "ALTER DATABASE myapp_test OWNER TO alex;" -U postgres
+- bundle exec rake db:migrate
+- gem install rubocop
```

Comment:
It's in the `Gemfile`, so you don't need to install it manually again. Just `bundle exec rubocop` should be enough